modules/apps/jaggery/emm/src/emm/modules/policy.js

```diff
@@ -85,10 +85,10 @@ policyModule = function () {
                 response.status = "success";
                 response.content = policyListToView;
                 return response;
-            },function(responsePayload){
+            }, function (responsePayload) {
                 var response = {};
                 response.status = "error";
-                if(responsePayload  == "Scope validation failed"){
+                if (responsePayload == "Scope validation failed") {
```

Comment:
This need to be here since the status code 401 occurs in four different scenarios such as.
Unauthorized.
Access Token Expired.
Access Token Invalid.
Scope validation failed.

In order to identify the current scenarios correctly comparing the status code is insufficient.